release notes/v0.38.1.md

```diff
@@ -0,0 +1,7 @@
+k6 v0.38.1 is patch release containing a bugfix!
+
+### Threshold expression containing reserved symbols would fail [#2512](https://github.com/grafana/k6/issues/2512)
```

Comment:
```suggestion
### Threshold sub-metric selectors containing reserved symbols would fail ([#2512](https://github.com/grafana/k6/issues/2512))
```

"threshold expressions" would probably more accurately refer to something like `p(99) < 500`